kolibri/content/management/commands/importcontent.py

```diff
@@ -145,7 +149,7 @@ def _transfer(self, method, channel_id, path=None, node_ids=None, exclude_node_i
                 # determine where we're downloading/copying from, and create appropriate transfer object
                 if method == DOWNLOAD_METHOD:
                     url = paths.get_content_storage_remote_url(filename, baseurl=baseurl)
-                    filetransfer = transfer.FileDownload(url, dest)
+                    filetransfer = transfer.FileDownload(url, dest, session=session)
```

Comment:
This is where a large portion of the 23x content download speedup comes from. We weren't re-using the session! So it was creating entirely new connections to the server for each file. Now it's "keep alive".

CC @lyw07 @aronasorman

Reply:
iiiinteresting - great catch